blueprints/ember-cli-blueprint-test-helpers/files/node-tests/nodetest-runner.js

```diff
@@ -2,7 +2,7 @@
 
 var glob = require('glob');
 var Mocha = require('mocha');
-var Promise = require('ember-cli/lib/ext/promise');
+var Promise = require('rsvp');
```

Comment:
this shouldn't be called `Promise`, because that would actually be `RSVP.Promise` (see https://github.com/tildeio/rsvp.js/#basic-usage)

I would suggest `var RSVP = require('rsvp');` similar to https://github.com/Turbo87/ember.js/commit/e1fb01efe95740a91d329cf7ff56bdb3ea64db68